File: problems/2148-minimum-number-of-moves-to-seat-everyone/minimum-number-of-moves-to-seat-everyone.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        seats.sort()
        students.sort()
        result = 0
        for i in range(len(seats)):
            if seats[i] != students[i]:
                result += abs(seats[i] - students[i])
            
        return result

File: problems/2148-minimum-number-of-moves-to-seat-everyone/minimum-number-of-moves-to-seat-everyone.js
/**
 * @param {number[]} seats
 * @param {number[]} students
 * @return {number}
 */
const minMovesToSeat = (seats, students) => {
    const sortSeats = seats.sort((a, b) => a - b);
    const sortStudents = students.sort((a, b) => a - b);

    let total = 0;
    for (let i = 0; i < seats.length; i++) {
        if (seats[i] !== students[i]) {
            total += Math.abs(seats[i] - students[i])
        }
    }
    return total;
};